src/mcode-pilot/components/TreatmentOptionsOutcomes/TreatmentOptionsOutcomes.scss

```diff
@@ -1,5 +1,7 @@
 $color-gray-light: #bbb;
 $color-gray-lighter: #eee;
+$color-blue: #0084b8;
+$color-blue-light: #ecf3f5;
```

Comment:
Probably a separate JIRA, but we should follow up with Dan Reeves to see how/if this fits in with our overarching style-guide. If it does, we can add it to our variables file accordingly and reference from there.

Reply:
Talked with Greg and made a JIRA